test/functional/core/startup_spec.lua

```diff
@@ -463,6 +464,17 @@ describe('user config init', function()
     eq(init_lua_path, eval('$MYVIMRC'))
   end)
 
+  it('loads .exrc from cwd', function()
+    local exrc_path = '.exrc'
```

Comment:
I put this in `cwd` here because I don't know how to spawn the neovim with specific cwd. I tried passing it through `args = {xhome}`, but it didn't work.

Reply:
let's separate the context and the expectation
```lua
describe('with existing .exrc in cwd'
   setup_context
   ...
   it 'loads .exrc'
   ...
```

Reply:
Done.